main.go

```diff
@@ -205,6 +205,12 @@ func InitFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&webhookOpts.CertDir, "webhook-cert-dir", "/tmp/k8s-webhook-server/serving-certs/",
 		"Webhook cert dir, only used when webhook-port is specified.")
 
+	fs.StringVar(&webhookOpts.CertName, "webhook-cert-name", "tls.crt",
+		"Webhook cert name, only used when webhook-port is specified.")
```

Comment:
> only used when webhook-port is specified

Is this true?

Reply:
Removed. The port nowadays defaults to 9443, so its always set and defaults to tls.